src/mon/PGMap.cc

```diff
@@ -4707,9 +4709,12 @@ int reweight::by_utilization(
   double expected_data_movement = (double) pgm.osd_sum.kb_used;
   expected_data_movement *= (double) pg_moved / (double) pg_total;
   if (f) {
-    f->dump_float("Expected data flow in KB: ", expected_data_movement);
+    f->dump_float("Expected data flow (in KB): ", expected_data_movement);
+    f->dump_int("Time for optization (in s): ", std::difftime(end, start));
```

Comment:
typo: optimization.

BTW, why would our user be interested in the time used to perform the re-weight? especially, it has already happened. if the user is really interested in time spend on reweight, he/she can just 
```
time ceph osd test-reweight-by-pg
```